docs/source/theming/index.md

```diff
@@ -31,6 +31,7 @@ theming-strategy
 ```{toctree}
 :maxdepth: 1
 
+../development/add-ons/theme
```

Comment:
I'm not sure we want to put this in the `toctree` here, as it can duplicate navigation trees. It also puts the same file in two locations. What would you like to do here? One option is a sphinx-reredirect.

Reply:
I think is important to have it side by side here and it is also valuable to have it in the add-ons docs too.
Your call we can move it from there to here too.